database/seeders/CompanySeeder.php

```
@@ -16,9 +16,11 @@ public function run(): void
     {
         $faker = Faker::create();
 
+        $index = 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$index'. (Unused Code Rules)

(UnusedLocalVariable)